### PR TITLE
Fix personalization click handler

### PR DESCRIPTION
**Pull request by jfkhoury**
## Description

We were causing a `click` collection call on every click because the Personalization was returning empty decisions meta data, and the event was considered not Empty.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Improvement (non-breaking change which does not add functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html) or I'm an Adobe employee.
- [x] I have made any necessary test changes and all tests pass.
- [x] I have run the Sandbox successfully.

### CODE REVIEW

File: src/components/Personalization/createOnClickHandler.js [hunk 1]
```diff
@@ -13,7 +13,10 @@ governing permissions and limitations under the License.
 export default ({ mergeMeta, collectClicks, clickStorage }) => {
   return ({ event, clickedElement }) => {
     const clickMetas = collectClicks(clickedElement, clickStorage);
-    event.mergeXdm({ eventType: "click" });
-    mergeMeta(event, { decisions: clickMetas });
+
+    if (clickMetas.length) {
```
Comment (ninaceban) [inline src/components/Personalization/createOnClickHandler.js:17]: I would use `isNonEmptyArray` util here.

Comment (artur-ciocanu) [reply src/components/Personalization/createOnClickHandler.js:17]: +1

Comment (jfkhoury) [reply src/components/Personalization/createOnClickHandler.js:17]: 👍

**Comment by garyr** [comment]
👍

**Pull request merged**